app/views/advanced_search_finder/show.html.erb

```diff
@@ -9,48 +9,52 @@
 <% end %>
 
 <%= render "govuk_publishing_components/components/inverse_header", { full_width: true } do %>
-  <div class="full-page-width-wrapper">
-    <div class="breadcrumb-wrapper">
```

Comment:
You've removed the `breadcrumb-wrapper` class, but it's still called in a css file (_advanced-search.scss)